Dnn.AdminExperience/Dnn.PersonaBar.Extensions/SqlDataProvider/03.00.00.SqlDataProvider

```diff
@@ -106,7 +106,13 @@ BEGIN
 						WHERE (LastName    Like @searchTerm)
 						  AND LastName  Is Not Null AND LastName  != N'' 
 						  AND (IsSuperUser = 1)
-						  AND (@isDeleted  Is Null OR IsDeleted = @isDeleted)) S 
+						  AND (@isDeleted  Is Null OR IsDeleted = @isDeleted)
+					   UNION SELECT UserID, Username, DisplayName, Email, CreatedOnDate, LastModifiedOnDate, IsDeleted, IsSuperUser
```

Comment:
This needs to be discussed for international date & large DB performance

Reply:
Also, if this is to be done, this should not be done here.  It should be done in a new `SqlDataProvider` file so it will fire for upgrades.

Reply:
@david-poindexter obviously, as stated before